ui_framework/components/table/_table.scss

```diff
@@ -63,13 +63,22 @@ $tableCellPadding: 7px 8px 8px;
 }
 
 .kuiTableRow {
+  transition: background $globalAnimSlightResistance $globalAnimSpeedSlow;
   &:hover {
     .kuiTableRowHoverReveal {
       display: inline-block;
     }
   }
 }
 
+.kuiTableRow-isBeingDeleted {
+  background: lighten($errorColor, 35%);
+}
+
+.kuiTableRow-isNew {
```

Comment:
I've been nesting these state classes inside of the root component alongside the pseudo-classes, since they're both used to communicate states. E.g. https://github.com/elastic/kibana/blob/master/ui_framework/components/tabs/_tabs.scss#L54

What do you think? Do you have a preference?